Bio/SeqIO/PdbIO.py

```diff
@@ -224,9 +224,10 @@ def iterate(self, handle):
             raise ValueError("Empty file.")
 
         for chn_id, residues in sorted(chains.items()):
-            record = SeqRecord(Seq("".join(residues), generic_protein))
-            record.annotations = {"chain": chn_id}
-            record.annotations["molecule_type"] = "protein"
+            record = SeqRecord(
+                Seq("".join(residues)),
+                annotations={"chain": chn_id, "molecule_type": "protein"},
```

Comment:
Same here.

Reply:
Because of the way PDB files work - this is the residue sequence.

Reply:
Except when it's RNA:
```
$ head 4gxy.pdb
HEADER    RNA                                     04-SEP-12   4GXY              
TITLE     RNA STRUCTURE                                                         
COMPND    MOL_ID: 1;                                                            
COMPND   2 MOLECULE: ADENOSYLCOBALAMIN RIBOSWITCH;                              
COMPND   3 CHAIN: A;                                                            
COMPND   4 ENGINEERED: YES                                                      
SOURCE    MOL_ID: 1;                                                            
SOURCE   2 SYNTHETIC: YES;                                                      
SOURCE   3 OTHER_DETAILS: IN VITRO TRANSCRIPTION                                
KEYWDS    RNA, RIBOSWITCH, ADENOSYLCOBALAMIN                                    
```
```python       
>>> from Bio import SeqIO
>>> record = SeqIO.read('4gxy.pdb', 'pdb-seqres')
>>> record.seq
Seq('GC')
>>> record.seq.alphabet
ProteinAlphabet()
>>> record.annotations
{'chain': 'A', 'molecule_type': 'protein'}
>>> 
```

Reply:
I was mistaken. We appear to have a pre-existing bug here when dealing with non-protein PDB files, where rather than no protein we can something strange.